mqtt_io/__main__.py

```diff
@@ -16,6 +16,8 @@
 from .modules import install_missing_requirements
 from .server import MqttIo
 
+_LOG = logging.getLogger('mqtt_io.__main__')
```

Comment:
Could this just use `__name__` instead of the string?

Reply:
__name__ would be '__main__', so you would need to setup a logger for 'mqtt_io' and '__main__' in the config file